docs/getting_started.md

```diff
@@ -171,12 +181,11 @@ apb build
 # run provision task
 docker run \
     --env "OPENSHIFT_TARGET=https://<oc-cluster-host>:<oc-cluster-port>" \
-    --env "OPENSHIFT_USER=admin" \
-    --env "OPENSHIFT_PASS=admin" \
+    --env "OPENSHIFT_TOKEN=<oc-token>" \
     <docker-org>/my-apb provision \
     --extra-vars 'namespace=getting-started'
 ```
-The environment arguments passed in using the `-e OPENSHIFT_TARGET`, `-e OPENSHIFT_USER`, and `-e OPENSHIFT_PASS`, will be used for the apb container to log in to the OpenShift cluster and are required.  The apb image tag `<docker-org>/my-apb` and action `provision` specify what to do.  The `--extra-vars 'namespace=getting-started'` is used to supply variables to the Ansible playbooks we'll write.  Running the above will output `'provision' NOT IMPLEMENTED`.
+The environment arguments passed in using the `-e OPENSHIFT_TARGET` and `-e OPENSHIFT_TOKEN`, will be used for the apb container to log in to the OpenShift cluster and are required.  The apb image tag `<docker-org>/my-apb` and action `provision` specify what to do.  The `--extra-vars 'namespace=getting-started'` is used to supply variables to the Ansible playbooks we'll write.  Running the above will output `'provision' NOT IMPLEMENTED`.
```

Comment:
Ah yes, you are right that you have to be logged in and you would essentially type `oc whoami -t`. It is equatable but I put that there because people were trying to run using the user/pass combo which the oc-login script no longer accepts.